app/src/main/java/com/cornellappdev/android/eatery/components/WaitTimesFragment.java

```diff
@@ -29,7 +30,11 @@
 
 public class WaitTimesFragment extends Fragment {
```

Comment:
now that i look at this, should this class really extend fragment? do you need lifecycle methods? if not, it can just be a view delegate class that is inside the `CampusMenuActivity` and we can hide/show the visibility of this view.

Reply:
Updated to a ComponentClass that inflates a view into a ViewGroup